apps/front-end/front-office/src/app/reducers/game/game.effects.ts

```diff
@@ -67,16 +87,31 @@ export class GameEffects {
   nextQuestion$ = createEffect(() =>
     this.actions$.pipe(
       ofType(GameActions.nextQuestion),
-      withLatestFrom(this.store.select(selectGameState)),
-      switchMap(([, state]) => {
-        const { quiz, currentQuestion } = state;
-        if (quiz) {
-          if (currentQuestion < quiz.questions.length) {
-            this.redirectToCorrectQuestion(quiz.questions[currentQuestion]);
-            return of(GameActions.nextQuestionSuccess());
-          }
+      withLatestFrom(
+        this.store.select(selectGameState),
+        this.store.select(selectGameCurrentQuestion)
+      ),
+      switchMap(([, state, currentQuestion]) => {
+        const { quiz } = state;
+
+        if (!quiz) {
+          return of(GameActions.endGame());
```

Comment:
should be an error

Reply:
https://github.com/2019-2020-ps6/2022-2023-ps6-webonjour/pull/198/commits/4c3b55fc5715dc9c8999275350b0e554cacb6edf